send_queue.go

```diff
@@ -64,7 +64,9 @@ func (h *sendQueue) Run() error {
 			shouldClose = true
 		case p := <-h.queue:
 			if err := h.conn.Write(p.Data); err != nil {
-				return err
+				if !isMsgSizeErr(err) {
```

Comment:
Can you add a comment here, explaining
1. That this makes it possible to run MTU discovery
1. That we're returning MTU errors from the kernel (and that the kernel might act upon ICMP packets)
1. That the packet will be detected lost by loss recovery (eventually)